vm/src/builtins/builtinfunc.rs

```diff
@@ -38,17 +38,20 @@ impl PyNativeFuncDef {
     pub fn build_function(self, ctx: &PyContext) -> PyObjectRef {
         self.into_function().build(ctx)
     }
-    pub fn build_method(self, ctx: &PyContext) -> PyObjectRef {
+    pub fn build_method(self, ctx: &PyContext, class: &PyTypeRef) -> PyObjectRef {
```

Comment:
`class: PyTypeRef`

when `class` is used by only clone, it doesn't need to be a reference.
(and also for callers)